core/src/main/java/org/mskcc/cbio/portal/util/GlobalProperties.java

```diff
@@ -770,6 +772,14 @@ public static String getDarwinAuthority() {
         return darwinAuthority;
     }
     
+    public static String getPriorityStudies() {
```

Comment:
Maybe either
  `Map<String, String> //study to category`
or
   `Map<String, Set<String>> //category to studies`
can be returned instead of `String`?